src/App.tsx

```diff
@@ -20,6 +35,11 @@ export class App extends React.Component<{}, State> {
 
               <div className="control">
                 <input
+                  onChange={(event) => {
+                    this.setState({ query: event.target.value });
+                  }}
```

Comment:
If callback doesn't require arguments other than `event` - move it to variable and use here as callback.